fronts-client/src/services/recipeQuery.ts

```diff
@@ -135,21 +144,21 @@ const recipeQuery = (baseUrl: string) => {
 			const queryString = args.length > 0 ? '?' + args.join('&') : '';
 			const url = `${baseUrl}/keywords/contributors${queryString}`;
 			const response = await fetch(url);
-			const content = await response.json();
+			const content = await response.text();
 			if (response.status == 200) {
-				return content as ChefSearchResponse;
+				return JSON.parse(content) as ChefSearchResponse;
```

Comment:
Is there an increased risk of json parse errors here? Should we wrap these in a try and catch the errors there too?

Reply:
In _principle_, no, because all 200 responses come from the (search) app and they are all definitely JSON (this is how it was working before, calling `await response.json()`).
On the other hand, it's not a bad idea just to be on the safe side.....

Reply:
Actually, no, I take that back; all of the _callers_ of these functions wrap in `try`/`catch` and report the error up that way, so we should not need to worry about it here.